test/cqlpy/cassandra_tests/validation/entities/vectors_test.py

```diff
@@ -0,0 +1,16 @@
+# This file was translated from the original Java test from the Apache
+# Cassandra source repository, as of commit 54e46880690bd5effb31116986292c1bdc9e891e
+#
+# The original Apache Cassandra license:
```

Comment:
> cassandra_tests: translate tests covering the vector type.
> 
> Added cql_vector_test which tests the basic functionalities of the vector type using CQL.
> Added vectors_test which tests if descending ordering of vector is supported.

Use imperative mood, please

Reply:
The commit message has been changed accordingly.